static/monorail-2.0.yaml

```diff
@@ -221,6 +247,24 @@ definitions:
             type: object
         type: object
     type: object
+  relations.2.0_Relation:
+    description: A key value set of relations.
+    properties:
+      containedBy:
+        description: The rack containing the node.
+        items:
+          type: string
+        minItems: 1
+        type: array
+        uniqueItems: true
+      contains:
+        description: An array of nodes contained by the rack.
+        items:
+          type: string
+        minItems: 1
+        type: array
+        uniqueItems: true
+    type: object
```

Comment:
I understand,  but I use the same schema for GET, I left them out [ and for now, the secondary schema validation will cover this case ]. With this change, we would have to re-visit to remove that one too , which is out of scope of this story . Possibly I can add two. I have to give some thoughts there.